core/frontend/src/components/logs/LogManager.vue

```diff
@@ -115,19 +115,24 @@ export default Vue.extend({
   methods: {
     async fetchAvailableLogs(): Promise<void> {
       const new_logs: FilebrowserFile[] = []
-      this.fetching_logs = true
 
       const log_folders = ['/root/.config/ardupilot-manager/firmware/logs/', '/root/.config/ardupilot-manager/logs/']
 
-      for (const folder_path of log_folders) {
-        await filebrowser.fetchFolder(folder_path)
-          .then((folder) => {
-            Array.prototype.push.apply(new_logs, folder.items)
-          })
+      // We fetch all paths in parallel and wait for everything to finish
+      // If it fails the folder does not exist, we display a 'No data available' message
```

Comment:
Where is the 'No data available' message logic?

Reply:
> Where is the 'No data available' message logic?

It's already on the list component I believe, on vuetify side

Reply:
Perfect. It's the `v-data-table` component behavior.